core/pom.xml

```diff
@@ -159,6 +159,13 @@
       <groupId>org.infinispan</groupId>
       <artifactId>infinispan-core</artifactId>
     </dependency>
+
+    <dependency>
+      <groupId>com.netflix.hystrix</groupId>
+      <artifactId>hystrix-core</artifactId>
+      <version>1.5.18</version>
+    </dependency>
```

Comment:
Was this an intentional addition? I didn't see where you used it...

Reply:
ups, that was left out from my experimenting with it :) , sorry , i will remove it.